<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDiffWidget</class>
 <widget class="QWidget" name="FileDiffWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="FileDiffSliderWidget" name="widget_diff_slider" native="true">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="FileViewWidget" name="widget_diff_left" native="true">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="FileViewWidget" name="widget_diff_right" native="true">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QScrollBar" name="verticalScrollBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="singleStep">
          <number>0</number>
         </property>
         <property name="pageStep">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QScrollBar" name="horizontalScrollBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="singleStep">
          <number>0</number>
         </property>
         <property name="pageStep">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="MaximizeButton" name="toolButton_fullscreen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/image/maximize.png</normaloff>:/image/maximize.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileDiffSliderWidget</class>
   <extends>QWidget</extends>
   <header>FileDiffSliderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MaximizeButton</class>
   <extends>QToolButton</extends>
   <header>MaximizeButton.h</header>
  </customwidget>
  <customwidget>
   <class>FileViewWidget</class>
   <extends>QWidget</extends>
   <header>FileViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
